src/com/yaji/Harness.java

```diff
@@ -0,0 +1,51 @@
+package com.yaji;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Harness {
+        private Date purchaseDate, lastDate;
+        private double  size;
+        private Integer gender;
+        private String brand;
+    private DateFormat dateFormat = new SimpleDateFormat("d.M.y");
+
+        public Harness(String purchase_date, String last_date, String harness_brand, double harness_size, Integer Gender) throws ParseException {
```

Comment:
By providing a String instead of a Date object, you give the Harness class the responsibility to parse the data. This means that if you are using the code in a different time-zone, with a different time-format, etc, this class have to handle a lot of date-time related parsing (which is usually quite a pain). It is therefore better to simply pass a Date-object and then handle the parsing somewhere else. 

It is good to think about responsibilities: Would you expect your harness to be able to parse time data?